modules/handlebarsUtils.js

```diff
@@ -28,19 +28,30 @@
     /**
      * Usage: {{labelFor 'fieldName' value="suffix"}}
      * Generates: for="field-name-suffix"
+     * Usage: {{labelFor 'fieldName[x].sub' value="demo" x=123}}
+     * Generates: for="field-name-123_sub-demo"
```

Comment:
You have -123_ on this example but \_abc\_ on the next example. Are you using underscores, dashes, or periods?

Reply:
No... I was wondering about it. The current behavior is when the value is a number, it returns a dash before the value (e.g. field-name-123), but if the value is a string, it returns an underscore before the value instead (e.g. field-name_abc")

Should I figure out how to make both use dashes?